runalltests.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/bash
+
+# Run all tests from the Stratis CI test suite
+# Current distro: Fedora (29)
+
+PRESTAGE=`pwd`
```

Comment:
should the variable name PRESTAGE be changed to RUNALLTESTS, as well?

Reply:
It is the pre-staging directory, so I think it isn't out-of-place.  (It's also shorter than "$RUNALLTESTS".)